src/gradleinttest.sh

```diff
@@ -52,10 +56,15 @@ gradleinttest () {
     fi
     pg_isready -h ${PGHOST} -p ${PGPORT}
     psql -U postgres -f ./db/create-data-repo-db
-    # required for tests
-    ./gradlew assemble
-    ./gradlew check --scan
-    ./gradlew ${test_to_run} --scan
+    if [[ "${test_to_run}" == "testPerf" ]]; then
+      export TEST_RUNNER_SERVER_SPECIFICATION_FILE="perf.json" 
+      ./datarepo-clienttests/gradlew run --args="configs/suites/BasicSmoke.json"
```

Comment:
What client library is this test using? I think we should build the client library locally (maybe already happening above somewhere?) and then point the Test Runner to use this version (e.g. with ORG_GRADLE_PROJECT_datarepoclientjar env var or a command line arg -Pdatarepoclientjar).

Reply:
I don't think we'll need to worry about this if we're just combining with the integration test run. I think the client library is build in the jade-data-repo code, and I was attempting to handle this in [this PR](https://github.com/DataBiosphere/jade-data-repo/pull/575). But, I don't think this PR is needed anymore.

Reply:
Okay, but even if the client library is already being built, I think you'll still need to point the Test Runner at the newly built JAR file. Otherwise it will just use whatever version is hard-coded in the build.gradle file (e.g. 1.0.39-SNAPSHOT).